org.lflang.lfc/src/org/lflang/lfc/Main.java

```diff
@@ -477,6 +477,7 @@ private void runGenerator(List<Path> files, Injector injector) {
             this.generator.generate(resource, this.fileAccess, context);
 
             exitIfCollectedErrors();
+            // print all other issues (not errors)
```

Comment:
This comment is a bit strange. It suggests that errors are somehow filtered out and not reported, which is probably not the case. Maybe just strike it?

Reply:
The line above will exit if there are any errors. So the line below is only executed if there are no errors, but we still want to print all other messages (mostly warnings)